config/webpackDevServer.config.js

```diff
@@ -94,7 +94,7 @@ module.exports = function(proxy, allowedHost) {
       // We do this in development to avoid hitting the production cache if
       // it used the same host and port.
       // https://github.com/facebookincubator/create-react-app/issues/2272#issuecomment-302832432
-      app.use(noopServiceWorkerMiddleware());
+      app.use(noopServiceWorkerMiddleware("/"));
```

Comment:
This is the change made based on this -> [article from StackOverflow](https://bit.ly/2U1gKfX)